src/community/jdbcstore/src/main/resources/jdbcstore.properties

```diff
@@ -44,4 +44,4 @@ pool.validationQuery=SELECT now()
 deleteDestinationOnRename=true
 
 # leave the following directories on the filesystem -- no spaces!
-ignoreDirs=data,jdbcstore,workspaces,layergroups,temp,tmp
+ignoreDirs=data,jdbcstore,workspaces,layergroups,temp,tmp,logs
```

Comment:
I know they _can_ be managed by resource store (I ran into that behavior; it mostly worked fine). I am more wondering if we _should_ reccomend / support doing so by default.
Since you set up both jdbcconfig and jdbcstore using very similar property files, and after import into jdbcstore the jdbcconfig.properties still remains on the file system, the fact that editing this file no longer has an effect could be misleading (especially since jdbcstore.properties will still be filesystem-only)